### PR TITLE
Create 2022-04-27-Taichi-Lang-Weekly.md

### PR DESCRIPTION
**Pull request by lucywsq**

### CODE REVIEW

**Comment by lucywsq** [comment]
PHAL. Thanks!

**Review by gingerkidney — APPROVED** [review]
LGTM

File: weekly/2022-04-27-Taichi-Lang-Weekly.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+[26 pull requests ](https://github.com/taichi-dev/taichi/pulls?q=is:pr+is:closed)were merged.
+
+* **Lang**
+    * [Add support for keyword arguments](https://github.com/taichi-dev/taichi/pull/4794)
+    * [Support in-place operator of ti.Matrix in python scope](https://github.com/taichi-dev/taichi/pull/4799)
+    * [Add 2D and 3D rotation functions to math module](https://github.com/taichi-dev/taichi/pull/4822) 
+
+* **Doc**
+    * [LLVM sparse runtime design doc](https://github.com/taichi-dev/taichi/pull/4790)
+* **CI**
```
Comment (ailzhang) [inline weekly/2022-04-27-Taichi-Lang-Weekly.md:20]: no need to include this one IMHO

File: weekly/2022-04-27-Taichi-Lang-Weekly.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    * [Use a new PAT with org permission](https://github.com/taichi-dev/taichi/pull/4826)
+* **Other**
```
Comment (ailzhang) [inline weekly/2022-04-27-Taichi-Lang-Weekly.md:22]: Misc

File: weekly/2022-04-27-Taichi-Lang-Weekly.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    * [[Error] Show warning when serialize = True is set on a struct for](https://github.com/taichi-dev/taichi/pull/4844) 
+    * [[SIMT] Add shfl_down_f32 intrinsic.](https://github.com/taichi-dev/taichi/pull/4819)
+    * [[bug] Fix bug that building with TI_EXPORT_CORE:BOOL=ON failed](https://github.com/taichi-dev/taichi/pull/4850)
```
Comment (ailzhang) [inline weekly/2022-04-27-Taichi-Lang-Weekly.md:25]: these fits more in the lang section instead of misc indeed.

**Review by ailzhang — APPROVED** [review]
LGTM!

**Review by ailzhang — APPROVED** [review]

**Pull request merged**